torch/distributions/gumbel.py

```diff
@@ -27,6 +27,7 @@ class Gumbel(Distribution):
 
     def __init__(self, loc, scale):
         self.loc, self.scale = broadcast_all(loc, scale)
+        self.euler_gamma_1p = 1.57721566490153286060
```

Comment:
Is this okay?

Reply:
It's cheaper to make this a module level constant
```py
_euler_gamma_1p = 1 + 0.57721566490153286060
```

Reply:
Sure. Thank you.